hack/prepare-e2e.sh

```diff
@@ -28,7 +14,13 @@ apiVersion: v1
 kind: List
 items:
 
-### Fix kube-dns ###
+### Fix kube-dns RBAC issues ###
+# Create a ServiceAccount for kube-dns to use
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: kube-dns
+    namespace: kube-system
```

Comment:
Added this here rather than creating the service in the travis.yml file.

Reply:
My reasoning behind the separation is so that we are not opinionated in our e2e test scripts about using minikube. It'd be ideal if the e2e tests could be run against *any* valid/functioning kubernetes cluster (this is more relevant when it comes to us running our tests on GCE/GKE).

Happy to accept this for now, but I think there'll be more refactoring of this stuff soon 😄

Reply:
Ah, got it. Well thanks for merging this anyway.